src/SolutionTemplate/UnoSolutionTemplate.WinUI.netcore/Server/Program.cs

```diff
@@ -20,10 +20,10 @@ public static void Main(string[] args)
 
 			app.UseAuthorization();
 
-$if$($UseWebAssembly$ == True)
+#if (UseWebAssembly)
```

Comment:
This needs to use the actual dotnet new symbol which is `WebAssembly`
```suggestion
//+:cnd:noEmit
#if (WebAssembly)
```

Reply:
@dansiegel This didn't work (at least in VSIX)

![image](https://user-images.githubusercontent.com/31348972/217205424-f7c8e024-f279-4de0-a2f1-1f39952c1e12.png)

It might be working from `dotnet new` but I haven't tested.

Reply:
yeah dotnet templates and VSIX have different ways they work so I'm not surprised